src/components/NoticeOfWithdrawal/RecordToBeWithdrawn.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <VcardTemplate id="record-to-be-withdrawn">
+    <template #icon>
+      mdi-file-document-outline
+    </template>
+
+    <template #title>
+      Record to be Withdrawn
+    </template>
+
+    <template #content>
+      <v-row
+        no-gutters
+        class="px-6 py-7"
+      >
+        <v-col
+          cols="12"
+          sm="3"
+          class="pr-4"
+        >
+          <strong>Filing Type</strong>
+        </v-col>
+        <v-col
+          id="withdrawal-filing-type"
+          cols="12"
+          sm="9"
+        >
+          {{ getFilingToBeWithdrawn().displayName }}
+        </v-col>
+      </v-row>
+
+      <v-row
+        no-gutters
+        class="px-6 py-7"
+      >
+        <v-col
+          cols="12"
+          sm="3"
+        >
+          <strong>Filing Number</strong>
+        </v-col>
+        <v-col
+          id="withdrawal-filing-number"
+          cols="12"
+          sm="9"
+        >
+          {{ filingToBeWithdrawn }}
+        </v-col>
+      </v-row>
+
+      <v-row
+        no-gutters
+        class="px-6 py-7"
+      >
+        <v-col
+          cols="12"
+          sm="3"
+        >
+          <strong>Effective Date and Time</strong>
+        </v-col>
+        <v-col
+          id="withdrawal-filing-date"
+          cols="12"
+          sm="9"
+        >
+          {{ getFormattedFilingDate() }}
+        </v-col>
+      </v-row>
+    </template>
+  </VcardTemplate>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import { Getter } from 'pinia-class'
+import { VcardTemplate } from '@/components/common'
+import { ApiFilingIF } from '@/interfaces'
+import { useFilingHistoryListStore } from '@/stores/filingHistoryListStore'
+
+  @Component({
+    components: {
+      VcardTemplate
+    }
+  })
+export default class RecordToBeWithdrawn extends Vue {
+    @Prop({ required: true }) readonly filingToBeWithdrawn!: string
+
+    @Getter(useFilingHistoryListStore) getFilings!: Array<ApiFilingIF>
+
+    getFilingToBeWithdrawn (): ApiFilingIF| undefined {
```

Comment:
Lint: add space before `|`.

Also we don't usually care about union to undefined or null but you can leave it.

Note: IMO, it's better to return null than undefined if "find" doesn't find it. This is because null properties are reactive but undefined ones are not. You can add `|| null` to line 93 to do this.